test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('assert');
+
+process.on("unhandledRejection", err => {
+  throw err;
+});
+
+// We try to import the generated module
+const features = require(__dirname+'/../dist/cjs');
+
+const isBoolean = function (value) {
+  return value === true || value === false;
+}
+
+const checkFeature = function (featureName) {
```

Comment:
Any reason why you didn't make this function async?

Reply:
Not any in particular. Will make it async!